mods/appmanager/src/client/appmanager.ts

```diff
@@ -82,13 +100,61 @@ export default class AppManager extends FonosService {
    * @todo if the file uploading fails the state of the application should
    * change to UNKNOWN.
    */
-  async deployApp(appPath: string, appRef?: string): Promise<App> {
+
+  async deployApp(appPath: string, appRef?: string): Promise<CreateAppRequest> {
```

Comment:
Perhaps we should follow the same pattern we have everywhere else in which we use a compound parameter instead of multiple parameters.a

Reply:
Solved on df73df98cddac282d057653ec910c2ae185b95ad